doc/manual/src/glossary.md

```diff
@@ -73,8 +73,7 @@
 
 - [fixed-output derivation]{#gloss-fixed-output-derivation}
 
-  A derivation which includes the
-  [`outputHash`](./language/advanced-attributes.md#adv-attr-outputHash) attribute.
+  A [derivation] where a cryptographic hash of the [output] is determined in advance using the [`outputHash`](./language/advanced-attributes.md#adv-attr-outputHash) attribute.
```

Comment:
```suggestion
  A [derivation] where a cryptographic hash of the [output] is determined in advance using the [`outputHash`](./language/advanced-attributes.md#adv-attr-outputHash) attribute.
  
  It trades flexibility in output for access to the network.
```

This is very significant. If not written here, it should be in the `outputHash` doc.

Reply:
Thought: its' a coderivation
```
              purity
            /        ^
           /          \
    derivation   coderivation
      realise      realise
           \          /
            v        /
             impurity

Reply:
```suggestion
  A [derivation] where a cryptographic hash of the [output] is determined in advance using the [`outputHash`](./language/advanced-attributes.md#adv-attr-outputHash) attribute, and where the [`builder`] executable has access to the network.